labgraph/examples/labgraph_monitor_example.py

```diff
@@ -0,0 +1,131 @@
+    SERIALIZER: Serializer
+
+    # Graph Components
+    NOISE_GENERATOR: NoiseGenerator
+    ROLLING_AVERAGER: RollingAverager
+    AMPLIFIER: Amplifier
+    ATTENUATOR: Attenuator
+
+    # Provide graph topology with `generate_labgraph_monitor()`
+    def set_topology(self, topology: SerializedGraph, sub_pub_map: Dict) -> None:
+        self._topology = topology
+        self._sub_pub_match = sub_pub_map
+
+    def setup(self) -> None:
+        self.WS_SERVER_NODE.configure(
+            WSAPIServerConfig(
+                app_id=APP_ID,
+                ip=WS_SERVER.DEFAULT_IP,
+                port=ENUMS.WS_SERVER.DEFAULT_PORT,
+                api_version=ENUMS.WS_SERVER.DEFAULT_API_VERSION,
+                num_messages=-1,
+                enums=ENUMS(),
+                sample_rate=SAMPLE_RATE,
+            )
+        )
+        self.SERIALIZER.configure(
+            SerializerConfig(
+                data=self._topology,
+                sub_pub_match=self._sub_pub_match,
+                sample_rate=SAMPLE_RATE,
+                stream_name=STREAM.LABGRAPH_MONITOR,
+                stream_id=STREAM.LABGRAPH_MONITOR_ID,
+            )
+        )
+        self.NOISE_GENERATOR.configure(
+            NoiseGeneratorConfig(
+                sample_rate=float(SAMPLE_RATE),
+                num_features=NUM_FEATURES,
+            )
+        )
+        self.ROLLING_AVERAGER.configure(
+            RollingConfig(
+                window=WINDOW,
+            )
+        )
+        self.AMPLIFIER.configure(
+            AmplifierConfig(
+                out_in_ratio=OUT_IN_RATIO,
+            )
+        )
+        self.ATTENUATOR.configure(
+            AttenuatorConfig(
+                attenuation=ATTENUATION,
+            )
+        )
+    
+    def connections(self) -> lg.Connections:
+        return (
+            (self.NOISE_GENERATOR.NOISE_GENERATOR_OUTPUT, self.ROLLING_AVERAGER.ROLLING_AVERAGER_INPUT),
+            (self.NOISE_GENERATOR.NOISE_GENERATOR_OUTPUT, self.AMPLIFIER.AMPLIFIER_INPUT),
+            (self.NOISE_GENERATOR.NOISE_GENERATOR_OUTPUT, self.ATTENUATOR.ATTENUATOR_INPUT),
+            (self.NOISE_GENERATOR.NOISE_GENERATOR_OUTPUT, self.SERIALIZER.SERIALIZER_INPUT_1),
+            (self.ROLLING_AVERAGER.ROLLING_AVERAGER_OUTPUT, self.SERIALIZER.SERIALIZER_INPUT_2),
+            (self.AMPLIFIER.AMPLIFIER_OUTPUT, self.SERIALIZER.SERIALIZER_INPUT_3),
+            (self.ATTENUATOR.ATTENUATOR_OUTPUT, self.SERIALIZER.SERIALIZER_INPUT_4),
+            (self.SERIALIZER.SERIALIZER_OUTPUT, self.WS_SERVER_NODE.topic),
+        )
+
+    def process_modules(self) -> Tuple[lg.Module, ...]:
+        return (
+            self.NOISE_GENERATOR,
+            self.ROLLING_AVERAGER,
+            self.AMPLIFIER,
+            self.ATTENUATOR,
+            self.SERIALIZER, 
+            self.WS_SERVER_NODE,
+        )
+
+if __name__ == "__main__":
+    graph = Demo()
+    topology, sub_pub_map = generate_graph_topology(graph=graph)
```

Comment:
probably we can add a level of abstract, and move lines 127, and 128 inside generate_graph_topology()

```
def generate_graph_topology(graph: lg.Graph) -> SerializedGraph:
    ...
    if hasattr(graph, "set_topology"):
        graph.set_topology(serialized_graph, sub_pub_map)  

    return serialized_graph, sub_pub_map
```